src/main/java/com/connexta/search/query/CommonQl.java

```diff
@@ -39,7 +38,7 @@
    * attributes and throw an exception if an unsupported attribute is detected.
    *
    * @return this for fluent interface
-   * @throws QueryException
+   * @throws IllegalQueryException
```

Comment:
@jhunzik I think you made the right decision to list the more specific exception (IllegalQueryException). I used QueryException because this method calls get g etFilter which throws a MalformedQueryException. QueryException is the superclass of both.

Reply:
Ah good catch, I think it's worth listing both exceptions. Right now we only use the super class for 500 type errors.